redbot/setup.py

```diff
@@ -287,7 +283,7 @@ def basic_setup():
                 if confirm("Would you like to import your data? (y/n) "):
                     await json_to_mongo(current_data_dir, storage_details)
         else:
-            storage_details = instance_data["STORAGE_DETAILS"]
+            storage_details = default_dirs["STORAGE_DETAILS"]
```

Comment:
NO NO NO. This should NOT have been changed. `storage_details` is set to the storage details from the instance settings for a very, very good reason (the actual storage details are needed if the user wants to import from MongoDB to JSON)

Reply:
That wasn't intended. Seems that's from another PR. Removing that...